src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -151,6 +154,16 @@ public String toString() {
         return persons.asUnmodifiableObservableList();
     }
 
+    /** Returns a list of persons at the provided indexes */
+    public ObservableList<Person> getPersonListByIndex(Set<Index> indexes) {
+        ObservableList<Person> personList = FXCollections.observableArrayList();
```

Comment:
Is there a need for the list to be observable if it's not going to be returned as unmodifiable?

(may want to return it as unmodifiable as indicate as such in the comments)